rs/cli/src/commands/neuron/top_up.rs

```diff
@@ -0,0 +1,31 @@
+use clap::Args;
+use ic_canisters::governance::GovernanceCanisterWrapper;
+
+use crate::commands::ExecutableCommand;
+
+#[derive(Args, Debug)]
+pub struct TopUp {}
+
+impl ExecutableCommand for TopUp {
```

Comment:
We also need a way to "refresh" the neuron account balance, otherwise the neuron will not known that there are more funds on the account.

> # It's also necessary to call the below function after sending funds to a neuron
> print("Refreshing neuron account balance (you will need to re-run this after getting more ICPs on the neuron wallet account)")
```
subprocess.check_call("dfx canister --network mainnet call governance manage_neuron '(record { id = opt record{ id = " + str(neuron_id) + " : nat64 }; command = opt variant { ClaimOrRefresh = record{ by = opt variant { NeuronIdOrSubaccount = record {} } } } })'", shell=True)
```

Reply:
Added, my bad